server/routes/service-provider/end-of-service-report/check-answers/endOfServiceReportCheckAnswersPresenter.test.ts

```diff
@@ -38,11 +38,9 @@ describe(EndOfServiceReportCheckAnswersPresenter, () => {
 
   describe('text', () => {
     it('returns text to be displayed', () => {
-      const presenter = new EndOfServiceReportCheckAnswersPresenter(
-        referral,
-        buildEndOfServiceReport(),
-        serviceCategory
-      )
+      const presenter = new EndOfServiceReportCheckAnswersPresenter(referral, buildEndOfServiceReport(), [
+        serviceCategory,
+      ])
```

Comment:
_nit_ this looks a bit hard to read, assuming it's the auto-formatter

Reply:
yeah, it's auto-formatter